src/qibo/backends/__init__.py

```diff
@@ -2,6 +2,9 @@
 from qibo import config
 from qibo.config import raise_error, log, warnings
 
+# versions requirements
+TF_MIN_VERSION = '2.2.0'
```

Comment:
We could also move this to config.py.

Reply:
Any special reason to move that to `config.py`?

Reply:
No, other than that we define all other "constants" used throughout the qibo code there. I don't have any strong preference on what is the right position for `TF_MIN_VERSION` though.